### PR TITLE
 Remove partials folder

### PR DESCRIPTION
**Pull request by nebelsprachen**

### CODE REVIEW

File: modules/ROOT/pages/gs-adding-custom-3scale-routes.adoc [hunk 1]
```diff
@@ -13,8 +13,8 @@ For example, you might want to expose an API as `https://mydomain.com/v1/getStuf
 To use a custom domain, you must create a route in the OpenShift 3scale project as described below.
 
 .Prerequisites
-* the `customer-admin` user credentials
-* a custom domain registration
+* The `customer-admin` user credentials
```
Comment (pwright) [inline modules/ROOT/pages/gs-adding-custom-3scale-routes.adoc:16]: what's the thinking here? these are not sentences

File: modules/ROOT/pages/gs-adding-users-proc.adoc [hunk 2]
```diff
@@ -50,9 +50,9 @@ Red Hat manages this instance, however the customer-admin user has admin level p
 |Red Hat Single Sign-On (Cluster)
 |Red Hat Single Sign-On
 
-When you click this link the first time an *Account already exists* message and an *Add to existing account* option are displayed. 
+When you click this link for the first time an *Account already exists* message and an *Add to existing account* option are displayed.
```
Comment (pwright) [inline modules/ROOT/pages/gs-adding-users-proc.adoc]:
```suggestion
When you click this link for the first time, an *Account already exists* message and an *Add to existing account* option are displayed. 
```

File: modules/ROOT/pages/gs-custom-keycloak-idp.adoc [hunk 3]
```diff
@@ -7,7 +7,7 @@ endif::[]
 = Adding custom identity providers to {PRODUCT}
 
 By default, {PRODUCT_SHORT} includes a Single Sign-On solution using Red Hat Single Sign-On (RHSSO).
-This lets users log in to all the middleware services using the same accounts.
+This allows a user to log in to all middleware services using the single account.
```
Comment (pwright) [inline modules/ROOT/pages/gs-custom-keycloak-idp.adoc]:
```suggestion
This allows a user to log in to all middleware services using a single account.
```

**Review by pwright — CHANGES_REQUESTED** [review]
have you run antora? what does it look like?
(i'm thinking of unresolved attributes)

File: modules/ROOT/pages/gs-adding-custom-3scale-routes.adoc (hunk 1 above)
Comment (nebelsprachen) [reply modules/ROOT/pages/gs-adding-custom-3scale-routes.adoc:16]: I thought, for consistency. The other prereqs are capitalized, but I guess it's not crucial.

**Review by pwright — APPROVED** [review]
approving just to figure out conflict

**Review by pwright — CHANGES_REQUESTED** [review]
park until next week

**Pull request closed without merging**